operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.1.0/ibm-spectrum-scale-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -243,8 +243,9 @@ spec:
   description:
     "\nThe IBM Spectrum Scale CSI Operator runs within a Kubernetes cluster
     providing a means to \ndeploy and manage the CSI plugin for spectrum scale. For
-    more in depth documentation please refer\nto the [README](https://github.com/IBM/ibm-spectrum-scale-csi-operator/blob/v1.1.0/README.md).\n\nThis
-    operator should be used to deploy the CSI plugin.\n\nThe configuration process
+    more in depth documentation please refer\nto the [github repository](https://github.com/IBM/ibm-spectrum-scale-csi) 
```

Comment:
Not super urgent... Are these lines generated, should we match the `\n` to each line for better readability? 

```
"\nThe IBM Spectrum Scale CSI Operator runs within a Kubernetes cluster providing a means to \n
deploy and manage the CSI plugin for spectrum scale.  For more in depth documentation please refer \n
to the [github repository](https://github.com/IBM/ibm-spectrum-scale-csi)  which has links to official documentation.\n\n
This operator should be used to deploy the CSI plugin.\n
The configuration process ....
```

Reply:
Lines are generated, we can fix it since the file it was generated from is no longer present.